src/components/ConfirmPage/ConfirmPage.container.ts

```diff
@@ -0,0 +1,18 @@
+import { connect } from 'react-redux'
+import { isConnecting } from 'decentraland-dapps/dist/modules/wallet/selectors'
+import { RootState } from '../../modules/reducer'
+import { validateSubscriptionEmailRequest } from '../../modules/subscription/actions'
+import HomePage from './ConfirmPage'
```

Comment:
do you mind renaming this import?

```suggestion
import ConfirmPage from './ConfirmPage'
```